optionals/tracers/CfgMagazines.hpp

```diff
@@ -0,0 +1,222 @@
+#define STRINGS(magazine) author = ECSTRING(common,ACETeam); displayName = CSTRING(magazine); descriptionShort = CSTRING(DOUBLES(magazine,description))
+
+class CfgMagazines {
+
+    // 5.56mm
+    class 150Rnd_556x45_Drum_Mag_F;
+    class ACE_150Rnd_556x45_Drum_green : 150Rnd_556x45_Drum_Mag_F {
+        ammo = "B_556x45_Ball_tracer_green";
```

Comment:
should also add
`displaynameshort="$STR_A3_tracer_dns";` to all tracers

Reply:
I do not think this is necessary. I checked Arma's config and all of the base classes I use have the correct displaynameshort configured, so it can be inherited.